script/frees_vs_max_k.rb

```diff
@@ -0,0 +1,117 @@
+# Print the Contest, Type (power vs glider), Category, and Pilot for any Free sequences
+# whose total figure K is less than the maximum allowable.
+#
+# Also flag any Frees whose figure K *exceeds* the allowable max... as long as we're at it. :-)
+#
+# DJM, 2020-10-18
+
+MAX_KS = {
+  'P' => {
+    'sportsman' => {
+      2006 => 127,
+      2007 => 132,
+      2008 => 135,
+      2009 => 125,
+      2010 => 137,
+      2011 => 127,
+      2012 => 124,
+      2013 => 149,
+      2014 => 136,
+      2015 => 130,
+      2016 => 126,
+      2017 => 133,
+      2018 => 121,
+      2019 => 119,
+      2020 => 129
+    },
+    'intermediate' => 190,
+    'advanced' => 300,
+    'unlimited' => 420
+  },
+  'G' => {
+    'sportsman' => {
+      2006 => 128,
+      2007 => 125,
+      2008 => 102,
+      2009 =>  94,
+      2010 => 113,
+      2011 => 124,
+      2012 => 127,
+      2013 => 127,
+      2014 => 106,
+      2015 => 117,
+      2016 =>  96,
+      2017 => 140,
+      2018 => 103,
+      2019 => 121,
+      2020 =>  94
+    },
+    'intermediate' => 140,
+    'advanced' => 175,
+    'unlimited' => 230
+  }
+}
+
+CATEGORIES = %w{ sportsman intermediate advanced unlimited }
+
+FREE_SEQ = 2  # JaSPer / IACCDB numbering convention for the Free program
```

Comment:
This is unused. Instead you used "where(name: 'Free')" in the query, which works because of:
```
mysql> select distinct name from flights;
+------------------+
| name             |
+------------------+
| Known            |
| Free             |
| Unknown          |
| Unknown 2        |
| Flight 2         |
| Team Unknown     |
| Flight 1         |
| Unknown II       |
| Flight 3         |
| Free Unknown     |
| Four Minute Free |
| Free Known       |
| Free Unknown 2   |
| Free Unknown 1   |
+------------------+
14 rows in set (0.03 sec)
```